Phonebook.Frontend/package.json

```diff
@@ -53,7 +53,7 @@
     "@ngx-translate/i18n-polyfill": "1.0.0",
     "@ngxs/devtools-plugin": "3.5.1",
     "@ngxs/logger-plugin": "3.5.1",
-    "@ngxs/router-plugin": "3.5.1",
+    "@ngxs/router-plugin": "^3.5.1-dev.master-faa521f",
```

Comment:
I know it is ugly but this commit will be released soon and is already in master. 
The other option would be to roll back to angular 7